orc8r/cloud/go/models/swagger-common.yml

```diff
@@ -151,3 +153,23 @@ definitions:
       message:
         type: string
         example: Error string
+  registration_info:
```

Comment:
add a helpful description please

Reply:
the other ones don't have a description and im slightly on a timeline if you wouldn't mind leaving as is (since generating takes a while) -- unless anything else should also be changed?